package.json

```diff
@@ -24,6 +24,7 @@
     "@shapeshiftoss/hdwallet-native": "^1.18.3",
     "@shapeshiftoss/hdwallet-native-vault": "^1.18.3",
     "@shapeshiftoss/hdwallet-portis": "^1.18.3",
+    "@shapeshiftoss/logger": "^1.0.3",
```

Comment:
I'm don't think this PR is the right to add a logger. We should talk about how we want to do logging separately.

Reply:
yeaaaah you're right changed this to a plain `console.info`